src/components/pure-component/hocs.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+function pure(func) {
```

Comment:
Whats the use of this?

Reply:
Looks like a helper function used to convert a functional component into a class component that extends React.PureComponent.  This is one of the files I ported over from MDVGridView.  I'm only using it once in BaseTable.js, so making BaseTable a pure component itself probably makes more sense.

Reply:
Its a functional component, so probably should leave it as such.